server/tests/backends/sql_translator_unit_tests/test_base_translator.py

```diff
@@ -126,10 +126,28 @@ def test_get_aggregate_function(base_translator: BaseTranslator, agg_type):
     assert issubclass(agg_func, functions.AggregateFunction)
 
 
-@pytest.mark.parametrize("agg_type", ["first", "last", "count distinct including empty"])
-def test_get_aggregate_function_raise_expection(base_translator: BaseTranslator, agg_type):
+@pytest.mark.parametrize("agg_type", ["first", "last"])
+def test__get_window_function(base_translator: BaseTranslator, agg_type):
+    agg_func = base_translator._get_window_function(agg_type)
+    assert issubclass(agg_func, functions.AggregateFunction)
+
+
+@pytest.mark.parametrize("agg_type", ["count distinct including empty"])
```

Comment:
can you add 'abs' agreggation here ? since we're not handling that for now !

Reply:
I don't see 'abs' in the available aggregations: 
![image](https://user-images.githubusercontent.com/71276735/178678493-c8b3f8fd-159b-4901-8dde-6afe086f0bf5.png)